src/algebra/field_power.lean

```diff
@@ -70,8 +70,8 @@ begin
     apply (inv_le_inv _ _).2,
     { apply pow_le_pow hx,
       have : -(↑(a+1) : ℤ) ≤ -(↑(b+1) : ℤ), from h,
-      have h' := le_of_neg_le_neg this,
-      apply le_of_coe_nat_le_coe_nat h' },
+      have h' : ↑(b + 1) ≤ ↑(a + 1) := neg_le_neg_iff.mp this,
+      exact le_of_coe_nat_le_coe_nat h' },
```

Comment:
What's the reason for this change? Did you remove `le_of_neg_le_neg` in this PR? I don't see where.

Reply:
I removed `le_of_neg_le_neg`, since the newer `neg_le_neg_iff` proves an iff version of the same statement but with weaker typeclasses.  If you think that I should restore it, I can do that.

Also, let me see if I can find the commit that removed it.

Reply:
Ah, I couldn't find it because I was should have been looking for `le_of_inv_le_inv `

Reply:
I think it's worth adding these removed lemmas back using `alias`; In this case `alias neg_le_neg_iff ↔ le_of_neg_le_neg _` should do the job.

Reply:
Even so, I am having trouble finding it: what do you tell git to find where the lemma disappeared?